addons/mail/static/src/messaging/entity/activity/activity.js

```diff
@@ -41,6 +41,14 @@ function ActivityFactory({ Entity }) {
             }
         }
 
+        /**
+         * @returns {boolean}
+         */
+        get isCurrentPartnerAssignee() {
+            return this.assignee && this.assignee.partner &&
+                this.assignee.partner.id === this.env.messaging.currentPartner.id;
```

Comment:
No need to compare ids, it should be the same object.
```suggestion
            return this.assignee && this.assignee.partner === this.env.messaging.currentPartner;
```